requirements/ci/py38.yml

```diff
@@ -25,7 +25,7 @@ dependencies:
   - graphviz
   - iris-sample-data >=2.4.0
   - mo_pack
-  - nc-time-axis >=1.3
+  - nc-time-axis >=1.4
```

Comment:
Not sure whether it's better to do this or to add some handling so we still use `CalendarDateTime` with nc_time_axis v1.3?

Reply:
I agree with the approach you have done here.
I don't see the benefit of adding support for older version nc_time_axis. There isn't a reason user's shouldn't be able to use the latest version of nc_time_axis (it is after all a small + simple package so it should bring dependency conflicts).